### ISSUE TITLE
SEUS Renewed creating major graphical glitches.

### ISSUE THREAD

**Issue by Prodabic** [iChun/IssueTracker-PortalGun#89]
I'm using version 7.1.0 on Windows 10 and playing singleplayer.
ForgeModLoader-client-0.log doesn't appear in my directory.

![2021-04-11_10 32 16](https://user-images.githubusercontent.com/26947025/114312698-33959380-9ab9-11eb-9084-c5f978d3751c.png)
![2021-04-11_10 31 25](https://user-images.githubusercontent.com/26947025/114312700-342e2a00-9ab9-11eb-817a-c28dfd42f295.png)
![2021-04-11_10 31 35](https://user-images.githubusercontent.com/26947025/114312701-342e2a00-9ab9-11eb-938a-b5c13ce5392a.png)

**Comment by iChun** [comment]
Can you try disabling stencils in the iChunUtil config and seeing if that
fixes it?

**Comment by Prodabic** [comment]
seems to be working now but there's still a minor visual glitch
![2021-04-13_15 25 33](https://user-images.githubusercontent.com/26947025/114616490-8a8f9a00-9c6c-11eb-825e-43bb353cd893.png)
![2021-04-13_15 26 03](https://user-images.githubusercontent.com/26947025/114616524-94b19880-9c6c-11eb-8264-c7d7667e8a47.png)
![2021-04-13_15 26 09](https://user-images.githubusercontent.com/26947025/114616527-954a2f00-9c6c-11eb-91e2-2815a6ed499b.png)

**Comment by iChun** [comment]
Yeah the shaders are busted as well, might have to turn them off, I don't
know if it's config based or if it uses the fast/fancy setting though.

**Comment by Prodabic** [comment]
alright
